showcase: add Alvaro Vanegas Site

ajvanegasv opened the pull request.

ajvanegasv committed:
add Alvaro Vanegas Site

hashrock commented on www/data/showcase.json:
```diff
@@ -271,6 +271,12 @@
+  {
+    "title": "Alvaro Vanegas Sites",
+    "link": "https://ajvanegasv.dev/",
+    "github": "https://github.com/ajvanegasv/my-portfolio",
```
`https://github.com/` is not needed

ajvanegasv replied:
resolved.

ajvanegasv committed:
Update showcase.json

This corrects the github property of the showcase.json file, the github domain is not required, only the name of the repository is needed.

hashrock committed:
Merge branch 'main' into main
